cm-mlops/automation/script/module.py

```diff
@@ -1716,6 +1723,16 @@ def _run(self, i):
         if verbose or show_time:
             print (recursion_spaces+'  - running time of script "{}": {:.2f} sec.'.format(','.join(found_script_tags), elapsed_time))
 
+
+        if not recursion and (verbose or show_space):
+            stop_disk_stats = shutil.disk_usage("/")
```

Comment:
Will this be a problem on huge NFS mounts?

Reply:
> Will this be a problem on huge NFS mounts?

Very good point. Could be! I changed this line and calculate space only if explicitly requested by --space flag. It's still a very useful feature when I test scripts for datasets and models - it's now very easy to see the used space consumed by the script ...